src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -338,7 +338,7 @@ private void BuildFallbackBlock(TagHelperContent builder)
 
                 // Build the <meta /> tag that's used to test for the presence of the stylesheet
                 builder
-                    .AppendHtml("<meta name=\"x-stylesheet-fallback-test\" class=\"")
+                    .AppendHtml("<meta property=\"x-stylesheet-fallback-test\" content class=\"")
```

Comment:
> There's a lot of things about this <meta> tag that is non-XHTML compliant. For instance ...

Interesting but not quite the same concern. Those are semantic issues. This attribute is invalid XML syntax. If we go with using the obscure open graph micro-format, why not use `content=""`?